make-util.js

```diff
@@ -1809,8 +1809,10 @@ exports.getBuildConfigGenerator = getBuildConfigGenerator;
  * @param {Object} makeOptions Object with all tasks
  * @param {Boolean} writeUpdates Write Updates (false to validateOnly)
  */
-var processGeneratedTasks = function(baseConfigToolPath, taskList, makeOptions, writeUpdates) {
+var processGeneratedTasks = function(baseConfigToolPath, taskList, makeOptions, writeUpdates, sprintNumber) {
     if (!makeOptions) fail("makeOptions is not defined");
+    if (sprintNumber && !parseInt(sprintNumber)) fail("Sprint is not a number");
```

Comment:
nit: for sprint number as "0" it will fail with that error but technically 0 is a legit number :)

Reply:
Totally forgot about it :) 
Replaced with Number.isInteger since [minimist](https://www.npmjs.com/package/minimist) cast arguments to correct types.
Thanks!